### ISSUE TITLE
Not work hopper dropped items

### ISSUE THREAD

**Issue by andris155** [TacoSpigot/TacoSpigot#19]
![2016-03-26_19 45 00](https://cloud.githubusercontent.com/assets/8324198/14061844/4e7f092c-f38b-11e5-89a1-aaa7bc5108c2.png)
![2016-03-26_19 45 04](https://cloud.githubusercontent.com/assets/8324198/14061845/53ff2044-f38b-11e5-9d0e-3543d9504528.png)

**Comment by Techcable** [comment]
version?

**Comment by Techcable** [comment]
do you have push-based set to true or false?

**Comment by andris155** [comment]
![940b644e-f387-11e5-899a-48e2485aa94c](https://cloud.githubusercontent.com/assets/8324198/14062572/6eef8fec-f3a4-11e5-8c57-763403f01f68.PNG)

**Comment by andris155** [comment]
Taco.yml
```
config-version: 1
world-settings:
  default:
    verbose: true
    armor-stand:
      optimize-movement: true
    hopper:
      fire-InventoryMoveItemEvent: true
      push-based: true
    redstone-fire-BlockPhysicsEvent: false
```

**Comment by Techcable** [comment]
does this occur if 'redstone-fire-BlockPhysicsEvent' is true?

**Comment by andris155** [comment]
yes

**Comment by andris155** [comment]
yes it

**Comment by andris155** [comment]
![2016-03-26_23 05 44](https://cloud.githubusercontent.com/assets/8324198/14062659/587c583c-f3a7-11e5-9350-91c50e705a33.png)

**Comment by Techcable** [comment]
does this occur if fire-InventoryMoveItemEvent' is false?

**Comment by andris155** [comment]
yes it

**Comment by andris155** [comment]
![2016-03-27_11 41 03](https://cloud.githubusercontent.com/assets/8324198/14064800/dac1a9a0-f410-11e5-9153-b4cade602698.png)

**Comment by jordanwilliams1** [comment]
This happend on my server to, It was to do with the taco hopper settings

**Comment by Techcable** [comment]
Hmm, I haven't had this issue in 1.8.
It must be an issue when I updated to 1.9.

**Comment by Techcable** [comment]
I think this is a paper issue.
Is it fixed in the [1.9.2 pre-release](https://github.com/TacoSpigot/TacoSpigot/releases/tag/v1.9.2-R0.1)?
